From e123c5e5f3cdeb820ae8ae697edb83600d094de6 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:06:37 -0400
Subject: [tools] perf: Show absolute percentage by default

Message-id: <1410181752-25631-153-git-send-email-jolsa@redhat.com>
Patchwork-id: 91863
O-Subject: [PATCH RHEL7.1 BZ1134356 152/307] perf tools: Show absolute percentage by default
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 95ce0ba17d5a0a04bbad61720512381d8165d157
Author: Namhyung Kim <namhyung@kernel.org>
Date: Tue Jan 14 12:05:27 2014 +0900

description
===========
Now perf report will show absolute percentage on filter entries by
default.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/symbol.c b/tools/perf/util/symbol.c
index b2eca6c..95e2497 100644
--- a/tools/perf/util/symbol.c
+++ b/tools/perf/util/symbol.c
@@ -33,7 +33,6 @@ struct symbol_conf symbol_conf = {
  .try_vmlinux_path = true,
  .annotate_src   = true,
  .demangle   = true,
- .filter_relative  = true,
  .symfs            = "",
 };
 
-- 
1.7.1